### PR TITLE
The Witness: Wake up honey it's another hints PR

### PR DESCRIPTION
**Pull request by NewSoupVi**
image

### CODE REVIEW

File: worlds/witness/hints.py [hunk 1]
```diff
@@ -180,9 +183,33 @@
...
+    "If you can't see the jungle for the trees, perhaps you should leaf.",
+    "Every puzzle has exactly 5 different solutions.\nThat doesn't seem right? That's because it's not!",
+    "Did you know you can turn off the audio logs by clicking on them again?",
+    "oOwOo *notices your audiolog*",
+    "This radon brought to you by ETAOIN SHRDLU",
+    "Please ignore the previous hint, that was meant for someone else. Networking is so difficult sometimes...",
+    "Oh, nice! I always forget to check this one.",
+    "Progressive Dots can be found. Probably. Unless they can't.",
+    "Don't know why the Mountain King, Mountain King, Mountain King,\nDon't know why the Mountain King has such a stressful halllll...",
+    "Fun Fact: If you laid out every line you've drawn end-to-end, then that would be weird. Why did you do that. That's not how this game works.",
```
Comment (ScipioWright) [inline worlds/witness/hints.py:195]:
```suggestion
    "Fun Fact: If you laid out every line you've drawn end-to-end, then that would be weird. Why did you do that? That's not how this game works.",
```

File: worlds/witness/hints.py [hunk 2]
```diff
@@ -180,9 +183,33 @@
...
+    "Sorry, this hint was removed by an Eraser.",
+    "I hear the Elevator is a super-handy shortcut if you forgot anything in the Tutorial area. :)",
```
Comment (ScipioWright) [inline worlds/witness/hints.py:197]:
```suggestion
    "I hear the Elevator is a super handy shortcut if you forgot anything in the Tutorial area. :)",
```
I don't think a dash here makes sense

**Review by ScipioWright — APPROVED** [review]

File: worlds/witness/hints.py (hunk 1 above)
Comment (NewSoupVi) [reply worlds/witness/hints.py]: Wait I just mindlessly clicked commit, I think this might be intentional

**Comment by NewSoupVi** [comment]
Made irrelevant by https://github.com/ArchipelagoMW/Archipelago/pull/3049

**Pull request closed without merging**